composables/useDownloadFile.ts

```diff
@@ -0,0 +1,24 @@
+export const useDownloadFile = () => {
+  const client = useSupabaseClient();
+
+  const download = (fileName: string) => {
+    const { data } = client.storage
+      .from("files/public")
+      .getPublicUrl(`${fileName}`, { download: true });
+
```

Comment:
I have useRetrievePublicUrl composable with getPublicUrl without {download:true}. Generally I wanted to call to supabase only once and handle a link for displaying file preview in the list and downloading a file, but some files eg. *.pdf were automatically downloaded and I think there were some other problems - this is why I decided to call to supabase separately for the preview link and the download link.